src/python/Cache/DocumentCache.py

```diff
@@ -0,0 +1,156 @@
+import os
+import json
+import csv
+
+from Cache.DocumentCacheBuilder import DocumentCacheBuilder
+
+
+class SSBProdStatusDocCache(DocumentCacheBuilder):
+    """
+    __SSBProdStatusDocCache__
+    General API for building the chaching document of key ssb_prod_status
+    """
+
+    def get(self) -> dict:
+        return None  # TODO: implement get_dashbssb() or drop this
+
+
+class SSBCoreMaxUsedDocCache(DocumentCacheBuilder):
+    """
+    __SSBCoreMaxUsedDocCache__
+    General API for building the chaching document of key ssb_core_max_used
+    """
+
+    def get(self) -> list:
+        return None  # TODO: implement get_dashbssb() or drop this
+
+
+class SSBCoreProductionDocCache(DocumentCacheBuilder):
+    """
+    __SSBCoreProductionDocCache__
+    General API for building the chaching document of key ssb_core_production
+    """
+
+    def get(self) -> list:
+        return None  # TODO: implement get_dashbssb() or drop this
+
+
+class SSBCoreCpuIntensiveDocCache(DocumentCacheBuilder):
+    """
+    __SSBCoreCpuIntensiveDocCache__
+    General API for building the chaching document of key ssb_core_cpu_intensive
+    """
+
+    def get(self) -> list:
+        return None  # TODO: implement get_dashbssb() or drop this
+
+
+class GWMSMONTotalsDocCache(DocumentCacheBuilder):
+    """
+    __GWMSMONTotalsDocCache__
+    General API for building the chaching document of key gwmsmon_totals
+    """
+
+    # TODO: url below does not return a json
+    def get(self) -> dict:
+        with os.popen("curl --retry 5 -s https://cms-gwmsmon.cern.ch/poolview/json/totals") as file:
```

Comment:
Hasan, it is in this file that gwmsmon is being used and in the WorkflowInfo ("https://cms-gwmsmon.cern.ch/prodview/json/{wf}/summary"). I included the usage list in here: #883.

@haozturk